packages/lesswrong/components/posts/PostsNewForm.tsx

```diff
@@ -230,12 +231,7 @@ const PostsNewForm = ({classes}: {
     <div className={classes.postForm}>
       <RecaptchaWarning currentUser={currentUser}>
         <Components.PostsAcceptTos currentUser={currentUser} />
-        {postWillBeHidden && <ContentStyles contentType="comment" className={classes.modNote}>
```

Comment:
Was this not forumGated  before?

Reply:
it was – the forumgating lived in the postWillBeHidden field.  (But, I think probably for this PR it'd probably be net positive for EA Forum to remove the forumGating, and showing the default "moderators will need to approve this" message. @jpaddison3 interested in your take)